inc/class-post-type.php

```diff
@@ -98,24 +98,4 @@ public static function has_archive( $post_type ) {
 	public static function has_metabox_enabled( $post_type ) {
 		return WPSEO_Options::get( 'display-metabox-pt-' . $post_type, false );
 	}
-
-	/**
-	 * Removes the notification related to the post types which have been made public.
-	 *
-	 * @return void
-	 */
-	public static function remove_post_types_made_public_notification() {
-		$notification_center = Yoast_Notification_Center::get();
-		$notification_center->remove_notification_by_id( 'post-types-made-public' );
-	}
-
-	/**
-	 * Removes the notification related to the taxonomies which have been made public.
-	 *
-	 * @return void
-	 */
-	public static function remove_taxonomies_made_public_notification() {
-		$notification_center = Yoast_Notification_Center::get();
-		$notification_center->remove_notification_by_id( 'taxonomies-made-public' );
-	}
```

Comment:
Should we not deprecate these instead?

Reply:
Let's keep them and defer the deprecation.